src/peripheral/mod.rs

```diff
@@ -91,6 +93,9 @@ mod test;
 #[allow(non_snake_case)]
 #[allow(clippy::manual_non_exhaustive)]
 pub struct Peripherals {
+    /// Cortex-M7 TCM and cache access control.
+    pub AC: AC,
```

Comment:
Perhaps the presence of this field in the struct should be cfg-gated too? And then also the declaration of the struct itself.

Reply:
Done